src/app/games/yinsh/yinsh.component.html

```diff
@@ -14,9 +14,9 @@
                          [ngClass]="getSpaceClasses(x, y)"
                          class="base"
                          stroke-linecap="round"/>
-                <g id="pieceGroup_{{ x }}_{{ y }}"
+                <g id="pieceGroup_{{ x }}_{{ y }}-SPACE_SIZE_IS{{SPACE_SIZE}}"
                    [ngClass]="getPieceGroupClasses(x, y)"
-                   [attr.transform]="getTranslationAt(getCenterAtXY(x, y))"
+                   [attr.transform]="'translate(' + getCenterAtXY(x, y).x + ' ' + getCenterAtXY(x, y).y + ')'"
```

Comment:
![image](https://github.com/user-attachments/assets/90887dbd-24a3-4fcf-a7a1-a259b61bea82)
is this regression ?

Reply:
c'était une régression du cul introduite oui !
J'suis passé à getSVGTranslationAt, mais c'est confusant as fuck que getTranslationAt prends des coordonnées abstraits et getSVGTranslationAt des concrets

Reply:
1. Si c'est dans le nom de la coord/x&y dans le paramètre de la méthode c'est déjà aidant, c'est le cas :D ?

2. Pour les jeux là, non affectés par ta PRRR, que fait-on ? Dans le sens ou ils mériteraient d'avoir un compteurs tous et que c'est le BEST TIME to make it
    BRANDHUB, HNEFATAFL, TABLUT: pièces restantes ? pièces capturées ?

    GO, TRI_GO:  techniquement c'est les captures en phase de jeux, puis le score en phase de comptage ?

    LASCA, INTERNATIONAL_CHECKERS: là aussi on utilise points mais c'est pas tso lo

    CONSPIRATEURS: un score de "pièces aux abris" ?
    DVONN: ?,
    GIPF: C'est des captures içi !,
    MARTIAN_CHESS: 'Points c'est bon, c'pas des captures ?',
    PYLOS: 'Pièces restantes ?',
    SQUARZ, REVERSI: pièces sur la plateaux ?,
    SIX: Pièces à déposer puis pièces restantes ?,

Reply:
1. maintenant oui !
2. je propose de noter cela dans le ticket game enhancements ? Mais oui, généraliser aux autres jeux serait bueno

Reply:
- tafl & co: pièces restantes
- go & trigo: captures puis points
- checkers: captures
- lasca: piles controllées
- Conspirateurs: pièces abritées
- Dvonn: ici on resterait sur points ?
- Gipf: captures
- MartianChess: captures oui
- Pylos: pièces restantes
- Squarz, reversi: points, simplement ?
- Six: pièces à déposer, pièces restantes